### PR TITLE
User system/user form pwdchange/issue 235

### PR DESCRIPTION
**Pull request by elfiyang16**
https://github.com/Greenstand/treetracker-admin/issues/235#issue-628300485

**create pwd change form** 
- currently show 3 different error message and 1 success message as shown below 
Jun-05-2020 21-47-34
Jun-05-2020 21-23-37
image

image

P.S i comment out line 76 in server/js/auth.js since I'm not sure what it's doing and I can't bypass it....happy to change back if required 
image

### CODE REVIEW

**Comment by dadiorchen** [comment]
@elfiyang16  This is great! 
About init(), this one is to avoid we deploy to a new environment and don't have initial data yet, it is kinda ugly but helpful for saving some trouble, for us, can I help you to bypass it? What's the problem with this?

File: server/src/js/auth.js [hunk 1]
```diff
@@ -192,6 +192,23 @@ router.get('/admin_users/', async (req, res, next) => {
   }
 });
 
+router.post('/validate/', async (req, res, next) => {
```
Comment (dadiorchen) [inline server/src/js/auth.js:228]: It's fine you use this path for the changing-password API

File: server/src/js/auth.js [hunk 2]
```diff
@@ -73,7 +73,7 @@ router.get('/permissions', async function login(req, res, next) {
 router.post('/login', async function login(req, res, next) {
   try {
     //try to init, in case of first visit
-    await init();
+    // await init();
```
Comment (dadiorchen) [inline server/src/js/auth.js]: If remove this, we will get some trouble when we deploy the system to a new environment without initial data in DB

**Comment by elfiyang16** [comment]
> @elfiyang16 This is great!
> About init(), this one is to avoid we deploy to a new environment and don't have initial data yet, it is kinda ugly but helpful for saving some trouble, for us, can I help you to bypass it? What's the problem with this?

Oh I see, thanks for clarification. So the problem is when we execute `/login` command for example, `init()` get executed, and when it found there's existing data in the db, it will simply console log`There are accounts in admin user, quit.` and return, so `login` never get executed @dadiorchen

**Comment by dadiorchen** [comment]
Can we uncomment the line of init()? Is there any trouble doing so? @elfiyang16

**Pull request merged**